app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,5 @@
+class DashboardController < ApplicationController
```

Comment:
@ikennaokpala , thank you for your work on this.

Why adding a new `DashboardController`?. The two filters together do not represent a Dashboard, this is just a temporary page until we have the final designs from Joe. 

Why not just add an action `filter` to the ContentItemsController, with a filter.html.erb and render there the Organisations filter?